app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import time
+from datetime import datetime
+
+
+def create_log_file() -> str:
+    current_time = datetime.now()
+    filename = f"app-{current_time.hour:02d}_{current_time.minute:02d}" \
+               f"_{current_time.second:02d}.log"
+    timestamp = current_time.strftime("%Y-%m-%d %H:%M:%S.%f")
+
+    with open(filename, "w") as file:
+        file.write(timestamp)
+
+    print(f"Created file: {filename}")
+    print(f"Timestamp: {timestamp}\n")
```

Comment:
It's not really what need's to be done. Read the task one more time and create the correct solution. You need to create a command which will allow to copy the file, not to create a log file